### PR TITLE
Add middle part for mobile portfolio website, Add the about-me, projects and experience sections

### PR DESCRIPTION
**Pull request by Lasten-Ernest**
This pull request is for the middle part of the mobile portfolio website. 
Added the about-me section, projects done so far and experience sections.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,57 +5,208 @@
...
-                            <a href="#"><img src="./assets/gitHubIcon.svg" alt="icon for GitHub"/></a>
-                            </li>
-                            <li class="media_links">
-                            <a href="#"><img src="./assets/peaceIcon.svg" alt="icon for M"/></a>
-                            </li>
-                        </ul>
-                    </div>
-                </div>
-            </main>
+  <div class="main_container">
```
Comment (danielufeli) [inline index.html:12]: - [ ] Nice working on your main `section`, kindly change this to a `main` HTML tag here for better use of semantics HTML

File: index.html [hunk 2]
```diff
@@ -5,57 +5,208 @@
...
-                            <a href="#"><img src="./assets/peaceIcon.svg" alt="icon for M"/></a>
-                            </li>
-                        </ul>
-                    </div>
-                </div>
-            </main>
+  <div class="main_container"> 
+    <div class="inside_container"> 
+      <section class="snapshots">
+        <div class="title-grid">
```
Comment (danielufeli) [inline index.html:15]: - [ ] Nice working on your project section, kindly re-align the cards so they match how it looks in the Figma Design you can view the below screenshot
- [ ] Kindly add a white `background` for the card, so it aligns with what you have in the Figma Template.

## Your Design
cc-1

## Figma Template
cc-2

File: index.html [hunk 3]
```diff
@@ -5,57 +5,208 @@
...
+            <li class="media_li">
+              <a href="#" target="_blank">
+                <img src="./assets/mIcon.svg" alt="M_Icon"/>
+              </a>
+            </li>
+          </ul>
+        </div>
+      </div> 
+      <a href="#" class="btn btn-primary">Get my resume</a>
+      <div class="languages">
```
Comment (danielufeli) [inline index.html:176]: - [ ]  Nice working on this `section` kindly re-align the frameworks and skills menu so it aligns with what you have in the Figma Template.

## Your Design
fig-2

## Figma Template
fig-1

**Review by danielufeli — CHANGES_REQUESTED** [review]
Hi @Lasten-Ernest, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linters error.
✔️ Github flow used properly.
✔️ Descriptive PR title & description.

## Required Changes ♻️
_Check the comments under the review._

- [ ] Kindly add back the header has this project is a continuation of what you did in your last project.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @danielufeli in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by zieeco — APPROVED** [review]
# status: APPROVED :green_circle: :100: 

Hi, @Lasten-Ernest 


68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f78543078657a514755357843444a754350652f67697068792e676966


Good job addressing the changes required by the previous reviewer 👍 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⚡ 
✅ Nice design 🔥 
✅ Linters are passing 👏 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**